packages/block-editor/src/components/block-tools/zoom-out-mode-inserters.js

```diff
@@ -33,7 +34,7 @@ function ZoomOutModeInserters() {
 		// eslint-disable-next-line @wordpress/data-no-store-string-literals
 		const editor = select( 'core/editor' );
 		return {
-			selectedSection: editor.getSelectedBlock(),
+			hasSelection: !! getSelectionStart().clientId,
```

Comment:
Why not switch to the correct store? Maybe it could use `getSelectedBlockClientId` instead? `getSelectionStart` will also work for multi selection, not sure if that's relevant here.

Reply:
Actually, it looks like it is relevant

Reply:
I've seen `hasSelection: !! getSelectionStart().clientId` pattern in a few places, so I also decided to use it here.

I don't have a strong preference for this and am happy to update.

Reply:
> Actually, it looks like it is relevant

What do you mean?

Reply:
I mean: the zoom out inserters should probably not be displayed if there's multi-selection either. Probably a bug rn.

Reply:
Should I switch the `getSelectedBlockClientId` or `getSelectedBlock` selector?

Reply:
I mean, that would be the minimal change, without changing functionality. But I think in this case it was also meant to catch multi-selection, so I would like it as you did.